kod/object/active/holder/nomoveon/battler/monster.kod

```diff
@@ -2964,8 +2964,8 @@ messages:
       
       if NOT test
       {
-         ptRandom=CreateTimer(self,@RandomTimer,
-                              viRandom_delay + Random(0,viRandom_delay/2));
+         ptRandom = CreateTimer(self,@RandomTimer,
+                                viRandom_delay + Random(0, (viRandom_delay * 3) / 2));
```

Comment:
Maybe simpler and easier to adjust, up to you: 
```
viRandom_delay * Random(2, 5) / 2
```

Reply:
Oh, love it! Thanks.

Reply:
Updated.